docs/en/observability/synthetics-create-test.asciidoc

```diff
@@ -174,6 +177,92 @@ For more details on getting started with the Synthetics Recorder, see <<syntheti
 
 image::images/synthetics-create-test-script-recorder.png[Elastic Synthetics Recorder after recording a journey and clicking Export]
 
+[discrete]
+[[synthetics-request-param]]
+== Make API requests
+
+You can use the `request` parameter to make API requests independently of the browser interactions.
+
+// Link to Playwright docs?
+// https://playwright.dev/docs/test-api-testing#sending-api-requests-from-ui-tests
+
+NOTE: The `request` parameter is not intended to be used for writing pure API tests. Instead, it is a way to support
+writing plain HTTP requests in service of a browser-based test.
```

Comment:
Waiting on confirmation from @andrewvc on this wording.

Reply:
perfect